### ISSUE TITLE
Issue with space between letters

### ISSUE THREAD

**Issue by AleksandrTabolin** [BigBadaboom/androidsvg#171]
Hi! I have a svg with text tag like this:
```
...
<text xmlns="http://www.w3.org/2000/svg" x="1.207" y="69.709" stroke="none" fill="#000000" font-family="'Arial'" font-size="2.822" id="DATID_STNR_48197641">автоматизированная МКПП</text>
...
```
And at result there are no spaces between letters:
![image](https://user-images.githubusercontent.com/11814784/58027016-65e22480-7b20-11e9-9866-fc08968c66c2.png)

**Comment by BigBadaboom** [comment]
Hi. Thanks for the report.

Can you attach the SVG so I can test it please?  Or at least post a version of the SVG with everything except the text removed.

**Comment by AleksandrTabolin** [comment]
<svg xmlns="http://www.w3.org/2000/svg"  width="100%" height="100%" viewBox="0 0 83.873 137.336" datversion="2.0" preserveAspectRatio="xMidYMid meet" zoomAndPan="magnify" version="1.0" contentScriptType="text/ecmascript" contentStyleType="text/css">  
  
  <g id="Hintergrund" fill-rule="evenodd" clip-rule="evenodd" stroke="#000000" stroke-linecap="round" fill="none"> 
    <g> 
      <polygon stroke="#FFFFFF" stroke-width="0.01" points="0.005 0.005 83.868 0.005 83.868 137.331 0.005 137.331 0.005 0.005"/> 
    </g> 
  </g>  
  <g id="Standardebene" fill-rule="evenodd" clip-rule="evenodd" stroke="#000000" stroke-linecap="round" fill="none"> 
   
    <g id="Text"> 
      <text x="1.207" y="69.709" stroke="none" fill="#000000" font-family="'Arial'" font-size="2.822" id="DATID_STNR_48197641">автоматизированная МКПП</text> 
    </g>  
   
  </g> 
</svg>

**Comment by BigBadaboom** [comment]
Fixed (sort of).

There are actually two issues here:

1. There's a bug in AndroidSVG where I needed to disable hinting when drawing text.  This is now fixed.  The fix will be in the 1.4 release.
2. There is a bug in Android from Lollipop to Oreo where font glyph spacing is wrong for fractional font sizes.  See: https://issuetracker.google.com/issues/37042531.
    This seems to be fixed now in Pie (28).  For the earlier versions of Android, the only workaround I know of is to not use fractional font sizes.  Eg. Change your SVG from `font-size="2.822"` to `font-size="3"`.

**Comment by AleksandrTabolin** [comment]
Thanks!

adding rounding did the job
```
        if (isSpecified(style, SVG.SPECIFIED_FONT_SIZE)) {
            float currentFontSize = getCurrentFontSize();
        
            style.fontSize.value = Math.round(style.fontSize.value);
        
            state.style.fontSize = style.fontSize;
            state.fillPaint.setTextSize(style.fontSize.floatValue(this, currentFontSize));
            state.strokePaint.setTextSize(style.fontSize.floatValue(this, currentFontSize));
        }
```
